sklearn/manifold/spectral_embedding_.py

```diff
@@ -301,7 +301,7 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         if embedding.shape[0] == 1:
             raise ValueError
 
-    elif eigen_solver == "lobpcg":
+    if eigen_solver == "lobpcg":
```

Comment:
As a side note, `codecov` is reporting that `lobpcg` is never used our tests:

https://codecov.io/gh/scikit-learn/scikit-learn/compare/92af3dabbb5f3381a656f7727171f332b8928e05...247e9c6029c37a68b279c91908f650a9c0173a39/src/sklearn/manifold/spectral_embedding_.py#L318